tool/ruby-openssl-docker/Dockerfile

```diff
@@ -63,24 +63,24 @@ RUN cd /build/openssl/libressl-2.4.4 && \
       --prefix=/opt/openssl/libressl-2.4 && \
     make && make install
 
-RUN curl -s http://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl-2.5.0.tar.gz | tar -C /build/openssl -xzf -
-RUN cd /build/openssl/libressl-2.5.0 && \
+RUN curl -s http://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl-2.5.4.tar.gz | tar -C /build/openssl -xzf -
+RUN cd /build/openssl/libressl-2.5.4 && \
```

Comment:
It looks like they have released new versions of LibreSSL 2.3 and 2.4 as well, according to http://www.libressl.org/releases.html.